tests/integration/conftest.py

```diff
@@ -277,6 +277,15 @@ def daemons_handler_module(request: pytest.FixtureRequest) -> None:
     yield from daemons_handler_implementation(request)
 
 
+@pytest.fixture(scope='module')
+def restart_wazuh_daemon_after_finishing_module(daemon: str = None) -> None:	
```

Comment:
This is only used for VD, move it to `tests/integration/test_vulnerability_detector/conftest.py`. Add a better description.

Reply:
Done in https://github.com/wazuh/wazuh/pull/18567/commits/a9d46de37527cd7de57335f479d99d19bff3c6d7